CMakeLists.txt

```diff
@@ -35,9 +35,13 @@ set(${PROJECT_NAME}_SRCS
   src/multi_library_class_loader.cpp
 )
 add_library(${PROJECT_NAME} SHARED ${${PROJECT_NAME}_SRCS})
-target_include_directories(${PROJECT_NAME}
-  PUBLIC include ${console_bridge_INCLUDE_DIRS} ${Poco_INCLUDE_DIRS})
-target_link_libraries(${PROJECT_NAME} ${console_bridge_LIBRARIES} ${Poco_LIBRARIES})
+if(ament_cmake_FOUND)
+  target_include_directories(${PROJECT_NAME}
+    PUBLIC include ${console_bridge_INCLUDE_DIRS} ${Poco_INCLUDE_DIRS})
+  target_link_libraries(${PROJECT_NAME} ${console_bridge_LIBRARIES} ${Poco_LIBRARIES})
+else()
+  ament_target_dependencies(${PROJECT_NAME} "console_bridge" "Poco")
```

Comment:
Does ament_target_dependencies support non-ament packages as dependencies?

Reply:
The API docs of the CMake function should answer the question: https://github.com/ament/ament_cmake/blob/207a07b72af1e5a780887d7c2268df57ef456af3/ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake#L19-L22